src/api/migrations/20190701162027-create-bookmark.js

```diff
@@ -0,0 +1,44 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('bookmarks', {
```

Comment:
the name of the table should start with a capital letter @truestbyheart

Reply:
please, find all the iterations of this and kindly correct it man

Reply:
Already done.